README.md

```diff
@@ -1,48 +1,74 @@
-Persistent Virtual Machine kubevirt add-on
+Persistent Virtual Machine KubeVirt add-on
 
-The persistent virtual machine (pvm) add-on adds a functionality to kubevirt
+The persistent virtual machine (pvm) add-on adds a functionality to KubeVirt
 to store a non running VMs in kubernetes.
 
-# Installing the add-on to vagrant deployment
+# Overview
 
-The command
+The persistence mechanism is based on the idea to use a separate resource to
+hold persisted VMs.
+The API is the same as for `VirtualMachines` just the `kind` is different.
+
+# Installation
+
+In order to enable persistent VMs, you need to define the new resource type in
+the cluster:
+
+```bash
+$ kubectl create -f manifests/persistentvm-resource.yaml
```

Comment:
Suggesting kubectl in this and every other instance instead of cluster/kubectl.sh is developer unfriendly decision. At the moment, if you

- install system from scratch and
- get kubevirt packages and sources

there is no `kubectl` on the host. Unless it is KubeVirt's dependency, it is bad idea to rely on it being present and needlessly complicate life of developers.